locales/it/json.json

```diff
@@ -331,17 +331,17 @@
     "If you did not receive the email": "Se non hai ricevuto l'email",
     "If you did not request a password reset, no further action is required.": "Se non hai richiesto un reset della password, non è richiesta alcuna azione.",
     "If you do not have an account, you may create one by clicking the button below. After creating an account, you may click the invitation acceptance button in this email to accept the team invitation:": "Se non hai un account, puoi crearne uno cliccando sul pulsante sotto. Dopo averlo creato, potrai cliccare il pulsante per accettare l'invito presente in questa email:",
-    "If you need to add specific contact or tax information to your invoices, like your full business name, VAT identification number, or address of record, you may add it here.": "If you need to add specific contact or tax information to your invoices, like your full business name, VAT identification number, or address of record, you may add it here.",
+    "If you need to add specific contact or tax information to your invoices, like your full business name, VAT identification number, or address of record, you may add it here.": "Se hai bisogno di aggiungere informazioni di contatto o fiscali specifiche alle tue fatture, come il nome completo dell'azienda, il numero di identificazione IVA o l'indirizzo di registrazione, puoi aggiungerle qui.",
```

Comment:
Lo cambierei in:

"Se hai bisogno di aggiungere informazioni di contatto o fiscali specifiche alle tue fatture, come il nome completo dell'azienda, il numero di partita IVA o l'indirizzo della sede legale, puoi aggiungerle qui."